.gitlab/e2e/include.yml

```diff
@@ -0,0 +1,9 @@
+
+---
+# e2e stage
+# Contains jobs which run new e2e tests for the Agent
+
+include:
+  - .gitlab/e2e/e2e.yml
+  - .gitlab/e2e/k8s_e2e.yml
+  - .gitlab/e2e/install_packages/include.yml
```

Comment:
This doesn't respect the one directory = one stage structure, should this be changed?

Reply:
I've aligned with the current convention, we might change it slightly in the future (we can make probably bigger chunks `lint`, `build`, `test`, `package`, `deploy`)